File: app/services/sql_service.py
def get_course_stats(_course_code: str) -> dict[str, object]:
    return {
        "average": "B-",
        "failRate": 0.12,
        "sampleSize": 200,
    }


def get_professor_ratings(name: str) -> dict[str, object]:
    return {
        "name": name,
        "overall": 4.2,
        "difficulty": 3.1,
        "wouldTakeAgain": 0.78,
        "sampleSize": 45,
    }

File: src/services/sql/sqlService.ts
import type { CourseStats, ProfessorRating } from "../../types";

export async function getCourseStats(_courseCode: string): Promise<CourseStats> {
  return {
    average: "B-",
    failRate: 0.12,
    sampleSize: 200,
  };
}

export async function getProfessorRatings(name: string): Promise<ProfessorRating> {
  return {
    name,
    overall: 4.2,
    difficulty: 3.1,
    wouldTakeAgain: 0.78,
    sampleSize: 45,
  };
}
